docs/interactions/Application_Commands.md

```diff
@@ -1,83 +1,130 @@
-# Slash Commands
+# Application Commands
 
-Slash Commands are the new, exciting way to build and interact with apps on Discord.
+Application commands are commands that an application can register to Discord. They generally have a name, description, and a set of options or arguments for a user to fill out. Unlike interactions, which can have dynamic data, application commands are guaranteed to be the same for any user that uses them.
 
-With Slash Commands, all you have to do is type `/` and you're ready to use your favorite bot. Users can learn everything your bot does and easily find new features as you add them. Validation, error states, and helpful UI walks them through your commands, meaning they can get it right the first time, especially on mobile. You now have one more ally in the fight against your phone's autocorrect.
+## Application Command Object
 
-Slash Commands set your users up for success instead of confusion and frustration. They separate how users think and how your code works, meaning no matter how complex your codebase and commands may get, people who love your bot will find it approachable and easy to use.
+> info
+> `CHAT_INPUT` command names must be all lowercase matching `^[\w-]{1,32}$`. `USER` and `MESSAGE` commands may be mixed case and can include spaces.
+
+###### Application Command Structure
 
-Let's get started!
+| Field               | Type                                                                                                                                           | Description                                                                | Valid Types |
+| ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------- | ----------- |
+| id                  | snowflake                                                                                                                                      | unique id of the command                                                   | all         |
+| type?               | one of [application command type](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-types)                | the type of command, defaults `1` if not set                               | all         |
```

Comment:
Do we get this property back on the interactions we receive from application commands now?

Reply:
Isn't this the application command data model? AFAICT from the example interactions, the only way to differentiate is to keep track of which command id is which type.

Reply:
We do - mentioned this in my review